.travis.yml

```diff
@@ -4,7 +4,6 @@ dist: focal
 language: go
 go:
 - 1.16.x
-- 1.15.x
```

Comment:
I think we do different tests based on 1.15 for now so might want to revert this one.

Reply:
Looks like just 

https://github.com/nats-io/nats-server/blob/4784205c2d60ee57f1c539e3eb5f19d03974a71d/.travis.yml#L32

I can update that too? Already go.mod declares 1.16 required so worth sorting it out

Reply:
We run different tests on each platform so we always have 2... Let's leave for now IMO. Agree be nice to remove but will want Ivan to weigh in.

Reply:
ok will revert